CONTRIBUTING.md

```diff
@@ -1,6 +1,28 @@
+Before reporting a bug
+---
+1. Search issue tracker for similar issues.
+2. Try the latest dev branch version of aframe and aframe-core.
```

Comment:
can you link to https://github.com/aframevr/aframe/tree/dev and https://github.com/aframevr/aframe-core/tree/dev here?